android/app/build.gradle

```diff
@@ -1,31 +1,62 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 21
-    buildToolsVersion "21.1.2"
+    compileSdkVersion 22
+    buildToolsVersion "22.0.1"
 
     defaultConfig {
         applicationId "com.crittercism"
         minSdkVersion 15
         targetSdkVersion 21
         versionCode 5
-        versionName "5.0"
+        versionName "5.3.1"
     }
+
     buildTypes {
+        ndkDebug {
```

Comment:
Just an FYI:  I think it would make sense to make the ndk build types "flavors" rather than "types".
Please don't change this now, it isn't worth our time at the moment. 

If ndk is a flavor then gradle will automatically create variants that are the cross product of the types & flavors (eg. ndk-debug, ndk-release, default-debug, and default-release).

I believe build flavors are generally for denoting which features are contained in the build (in this case NDK), whereas types are more for changing compiler settings.